spec.bs

```diff
@@ -3786,20 +3795,67 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|configIDL|)</dfn> m
+        : [=auction data per seller result/seller=]
+        :: |seller|
+        : [=auction data per seller result/error=]
+        :: "invalid coordinator"
+      1. [=list/Append=] |result| to |results|.
+      1. [=iteration/Continue=].
+    1. Let |keyInfo| be the result of [=looking up the server encryption key=]
+       with |config|'s [=auction data config/seller=] and |seller|'s [=auction data config/coordinator=].
+    1. If |keyInfo| is failure:
+          1. Let |result| be a new [=auction data per seller result=] with the following [=struct/items=]:
+            : [=auction data per seller result/seller=]
+            :: |seller|
+            : [=auction data per seller result/error=]
+            :: "key lookup failed"
+        1. [=list/Append=] |result| to |results|.
+        1. [=iteration/Continue=].
+    1. Let (|key|, |keyId|) be |keyInfo|.
+    1. Set |config|'s [=auction data config/encryption key=] to |key|.
+    1. Set |config|'s [=auction data config/encryption key id=] to |keyId|.
+    1. Let (|requestBlob|, |context|) be the result of serializing |igMap| with |config| and
+      |igPAggCoordinatorMap|. The serialization method may follow that described in
+      [Section 2.2.4 of Bidding and Auction Services](https://privacysandbox.github.io/draft-ietf-bidding-and-auction-services/draft-ietf-bidding-and-auction-services.html#name-generating-a-request).
+    1. Let |result| be a new [=auction data per seller result=] with the following [=struct/items=]:
+        : [=auction data per seller result/seller=]
+        :: |seller|
+        : [=auction data per seller result/request=]
+        :: |requestBlob|
+    1. [=list/Append=] |result| to |results|.
+    1. Let |requestContext| be a new [=server auction request context=].
+    1. Set |requestContext|'s [=server auction request context/request ID=] field to |requestId|.
+    1. Set |requestContext|'s [=server auction request context/request context=] field to |context|.
+    1. [=map/Set=] |global|'s [=associated Document's=] [=node navigable's=]
+      [=traversable navigable's=] [=traversable navigable/saved Bidding and Auction request context=][|requestId|] to |requestContext|.
+  1. Let |IDLresults| be a new {{AdAuctionData}}.
+  1. [=map/Set=] |IDLresults|["{{AdAuctionData/requestId}}"] to |requestId|.
+  1. If |configIDL|[{{AdAuctionDataConfig/seller}}] [=map/exists=]:
+    1. Let |seller result| be |results|[0].
+    1. If |seller result|'s [=auction data per seller result/error=] is not null:
+      1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to
+         [=reject=] |p| with a {{TypeError}}.
+      1. Abort these steps.
+    1. Set |IDLresults|["{{AdAuctionData/request}}"] to |requestBlob|.
+  1. Otherwise:
+    1. [=list/For each=] |seller result| in |results|:
+      1. Let |IDLresult| be a new {{AdAuctionPerSellerData}} whose {{AdAuctionPerSellerData/seller}}

```

Comment:
{{AdAuctionPerSellerData/seller}} is string, while the infra one is [=origin=]. Need to do [=serialization of an origin|serialization=] ([example](https://github.com/WICG/turtledove/blob/main/spec.bs#L1832))

Reply:
Done.